### ISSUE TITLE
czechhq.net (used to work until yesterday)

### ISSUE THREAD

**Issue by raymonders** [reek/anti-adblock-killer#2946]
Yes it's porn. But the AAK worked fine until yesterday. Have tried manually inputting the site url into the filter list, but does not fix issue.

**Comment by uBlock-user** [comment]
@X01X012013 Needs a fix for the popups and the anti adblock

**Comment by jspenguin2017** [comment]
Well, the rule for pornhub shuts down the popups, that was easy... 
For anti-adblock, I need to look into it when I have time. I see that anti-FuckAdBlock is triggered. If you can pin down the offending function, that would help a lot.

**Comment by jspenguin2017** [comment]
BTW, the domain for the video is `29443kmq.video` which kind of worry me a bit since this domain looks pretty temporary...

**Comment by uBlock-user** [comment]
@X01X012013 That makes a lot more easy for me. For anti adblock - set `xRds` to false and `cRAds` to true
Additionally block - `||datoporn.com/css/main.css`

**Comment by uBlock-user** [comment]
Another thing, whitelist `XHR` requests from `37.187.130.74` as it's blocked by Adguard English Filter and Easylist + Fanboy Ultimate Block list for domain `29443kmq.video`. This is for those who're subscribed to those two lists.

**Comment by jspenguin2017** [comment]
This is now fixed. @uBlock-user Thanks for your help.

**Comment by raymonders** [comment]
Thanks, I guess? Do simply update AAK now to fix the issue? Or must I apply your changes manually?

**Comment by raymonders** [comment]
Tried clicking on random video links. Not working 2/3 of the time:
problems seen:
video box is greyed out, with no message
or
video box is white with "disable adblock" message
or 
play button appears, but clicking does not start video

**Comment by jspenguin2017** [comment]
Sometimes the video that should be replacing the gray box doesn't load, but the player right below it works. I can't remove the gray box since sometimes it works. Out of a couple random links I clicked, at least one video player works. If you have a specific test link where nothing works, please post it.

**Comment by raymonders** [comment]
Here's one instance where, on my chrome browser, the top one is gray box, the bottom has disable adblock message
![example](https://cloud.githubusercontent.com/assets/21112692/23878460/ccf08a20-0847-11e7-8a23-f00e20418531.JPG)

and sometimes it's like this

![exemple2](https://cloud.githubusercontent.com/assets/21112692/23878483/eb6ae5ea-0847-11e7-899f-4d103e0a4d9b.JPG)

**Comment by jspenguin2017** [comment]
Where is your script host?

**Comment by uBlock-user** [comment]
First, Remove AAK script, second, AdBlock Protector Script is only compatible with uBlock Origin, so remove Adblock extension too, third you need to install script host like Tampermonkey or ViolentMonke yand then install the script itself.

**Comment by raymonders** [comment]
@X01X012013 my script host is tampermonkey, the toolbar just hides it
@uBlock-user i've followed your 3 steps, uninstalled AAK script and AdBlock Protector script. What script should I be installing in step 3?

edit: Ive intalled uBlock Script and List, but ads on this site are now ALL active, and at least 2 pop-ups whenever I click a link... The problem has gotten worse.

**Comment by jspenguin2017** [comment]
So in your extension list, you should have uBlock Origin and Tampermonkey. 
In uBlock Origin, do 2 changes from default settings: Tick "Adblock Warning Removal List" and subscribe to https://github.com/X01X012013/AdBlockProtector/raw/master/AdBlockProtectorList.txt
In Tampermonkey, you should have 1 script: https://github.com/X01X012013/AdBlockProtector/raw/master/AdBlockProtector.user.js

**Comment by raymonders** [comment]
@X01X012013 Thank you for the quick reply!
I assume that subscribing to the uBlock List is done by pasting the url into the "My filters" box? I have done the other 2 steps.

**Comment by jspenguin2017** [comment]
@raymonders No, click 3rd-party filters, scroll to the very bottom, paste in the textbox there.

**Comment by raymonders** [comment]
@X01X012013 oops, sorry! There,  ive done as instructed. ive also restarted my browser. However, the initial problem has returned. (i have adblock for youtube still enabled, does that create extension conflict?)

![still](https://cloud.githubusercontent.com/assets/21112692/24021377/e7223032-0aa0-11e7-9fbb-c3e697e0e25e.JPG)

**Comment by jspenguin2017** [comment]
@raymonders OK, it's working in Opera but not in Chrome... Let me check...

**Comment by raymonders** [comment]
@X01X012013 it seems quite random. on some links the openload (upper) box works but the lower box doesnt. in others it's the openload that's black and the lower one works right away. but some, like the link above, is always greyed out for both boxes. please take your time with this issue, it seems like something deep in the site's hosting codes.

**Comment by jspenguin2017** [comment]
@uBlock-user Oh no, they are watching us... Popunder code updated on 03.14
```html
<!-- PopAds.net Popunder Code for datoporn.com | 2017-03-14,1666808,0,0 -->
<script type="text/javascript" data-cfasync="false">
/*<![CDATA[/* */
 (function(){ var o=window;o["\x5f\x70\u006f\u0070"]=[["\u0073i\x74\x65I\x64",1666808],["\u006di\u006eB\x69\x64",0],["\x70op\u0075\u006e\u0064\x65\x72s\x50\x65\u0072I\x50",0],["\x64\x65\x6ca\x79B\x65\x74\x77\x65\x65\u006e",0],["\x64\u0065\x66a\u0075\u006ct",false],["de\u0066\u0061u\u006ctP\x65\u0072\x44\u0061y",0],["\u0074\u006f\x70\x6d\x6f\u0073\u0074\x4c\u0061\u0079\u0065\x72",!0]];var a=["\u002f\x2fc\u0031\u002e\x70\x6fp\u0061\u0064s\u002en\x65\u0074\u002f\x70\x6f\u0070.j\u0073","\x2f/\u0063\u0032\x2e\u0070\x6f\x70\x61\u0064s.\x6e\u0065t\x2fp\u006f\u0070\u002ej\x73","\x2f\x2f\x77w\u0077\x2e\u0074gi\x6ao\u0065\x7avmv\u0076\x6c.\u0063\x6fm\u002fl\x76\x74f\u002ej\u0073","\x2f\u002f\x77\u0077\x77\x2e\u0062sn\x62\u0066\x75\u0066\u006a\x67\x78\u0072\u0062.\u0063\x6fm\x2f\x67md\u002e\x6a\u0073",""],n=0,z,y=function(){if(""==a[n])return;z=o["\x64\u006fc\x75me\u006e\u0074"]["\x63\x72e\u0061t\u0065\u0045\x6cem\x65\u006e\x74"]("\x73\x63\u0072\u0069p\x74");z["t\x79\u0070\x65"]="t\u0065\u0078\u0074\u002f\x6a\x61\u0076as\u0063r\u0069p\u0074";z["\u0061\u0073\x79\x6e\x63"]=!0;var p=o["d\x6fcu\x6d\u0065\x6et"]["\u0067\u0065\x74E\x6ce\x6de\u006e\u0074sB\u0079Ta\x67\u004e\x61\x6d\x65"]("\x73\x63\u0072ip\x74")[0];z["\x73\x72\x63"]=a[n];if(n<2){z["c\x72oss\u004f\x72ig\u0069n"]="\u0061n\x6f\u006ey\x6do\x75\x73";};z["o\x6e\x65r\u0072or"]=function(){n++;y()};p["p\u0061\u0072en\u0074\u004eo\x64\u0065"]["\x69ns\x65\u0072\x74\x42\u0065f\x6fr\x65"](z,p)};y()})();
/*]]>/* */
</script>
```

**Comment by jspenguin2017** [comment]
@raymonders If you right click on the block screen and click reload frame, it might work. The new popunder is kind of a pain to deal with... Still looking for solution.

**Comment by uBlock-user** [comment]
@X01X012013 https://github.com/X01X012013/AdBlockProtector/issues/93#issuecomment-287319816

PopAds is patched by uBO, and kill all third party script except from `datoporn.com` and `29443kmq.video`, the website is swarming with third party shitware.
